### ISSUE TITLE
 [Feature]: Locate playing song in the player queue

### ISSUE THREAD

**Issue by RABINGG** [fast4x/RiMusic#1795]
### Description

Option to locate the current playing song in the player queue. Now scrolling up and down to find the playing 

### Suggested Solution

Provide the same locate option in Playlist in player queue also

**Comment by fast4x** [comment]
in queue there is a musicbar animated icon...

**Comment by RABINGG** [comment]
> in queue there is a musicbar animated icon...

Yes I know that but for a queue of 100+ songs Arranging the queue and then finding the now playing song using the little animation in the small thumbnail is difficult. Maybe a change in animation thats easy to spot or a locate icon could help
